core/templates/components/version-diff-visualization/version-diff-visualization.component.ts

```diff
@@ -309,7 +313,7 @@ export class VersionDiffVisualizationComponent implements OnInit {
     this.legendGraph = {
       nodes: {},
       links: []
-    };
+    } as LegendGraph;
```

Comment:
Just remove typecasting from here.

Reply:
done

Reply:
in the whole codebase we have used as `typecasting` I think if it is bad practice can you please make an issue for the same.